src/typings.d.ts

```diff
@@ -37,11 +37,17 @@ declare let ANT_DESIGN_PRO_ONLY_DO_NOT_USE_IN_YOUR_PRODUCTION: 'site' | undefine
 
 declare const REACT_APP_ENV: 'test' | 'dev' | 'pre' | false;
 
-declare let ADMIN_API_SCHEMA; let ADMIN_API_HOST; let ADMIN_API_PATH; let API_KEY: '';
+declare let ADMIN_API_SCHEMA;
+declare let ADMIN_API_HOST;
+declare let ADMIN_API_PATH;
```

Comment:
I'm not sure why ADMIN_API_PATH doesn't have initial data, but API_KEY has..

Reply:
This is just a definition to help the IDE know the variable type.